src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,123 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+
+    //  Initialize array of adjacents of each tomato
+    let adjacents = [];
+    //  Initialize grid of adjacents according to the grid of tomatoes
+    let adjacentsArr = []
+    //  Initialize queue to use for Breadth First Search
+    let queue = []
+    //  Initialize count to track number of minutes
+    let count = 0
+
+    //  Generate a grid of adjacent positions of each tomato
+    for (let row = 0; row < grid.length; row++){
+        let rowAdjacentArr = []
+        for (let column = 0; column < grid[row].length; column++){
```

Comment:
You use the `grid.length` and `grid[row].length` in quite a few places, you could extract them out to variables.

Reply:
Noted sir.